src/Main.elm

```diff
@@ -22,9 +22,8 @@ init flags =
         authors =
             Decoders.authors flags.authorsList
 
-        { affiliationLimit, authorLimit, showInstitution, showCity, showState, showCountry, mandatoryInstitution,
-          mandatoryCity, mandatoryState, mandatoryCountry, class, institutionLabel, cityLabel, stateLabel, countryLabel}
```

Comment:
Why are we getting rid of the affiliation labels?

Reply:
looks like funny merge business